modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -237,7 +237,6 @@ export default class Checkbox extends React.Component<CheckboxProps> {
   static ErrorType = ErrorType;
 
   public static defaultProps = {
-    checked: false,
```

Comment:
Interested in opinions on this. To properly support `defaultChecked`, I had to remove `checked` as a default prop so that it is initially `undefined`. Otherwise, `defaultChecked` would always override `checked` whenever it is false.

Reply:
@anicholls do we want to have the same behavior for radio as well?

Reply:
Yeah - we should. Maybe this should be done in a separate PR. Thoughts @lychyi?

Reply:
Ended up removing this! Ignore 🙂